doc/transform_methods/linbasex.rst

```diff
@@ -0,0 +1,156 @@
+.. |nbsp| unicode:: 0xA0 
+   :trim:
+
+Lin-Basex
+=========
+
+
+Introduction
+------------
+
+Inversion procedure based on 1-dimensional projections of VM-images as 
+described in Gerber et al. [1]. 
+
+[ *from the abstract* ]
+
+*VM-images are composed of projected Newton spheres with a common centre. 
+The 2D images are usually evaluated by a decomposition into base vectors each
+representing the 2D projec- tion of a set of particles starting from a centre 
+with a specific velocity distribution. We propose to evaluate 1D projections of
+VM-images in terms of 1D projections of spherical functions, instead. 
+The proposed evaluation algorithm shows that all distribution information can 
+be retrieved from an adequately chosen set of 1D projections, alleviating the 
+numerical effort for the interpretation of VM-images considerably. The obtained
+results produce directly the coefficients of the involved spherical functions, 
+making the reconstruction of sliced Newton spheres obsolete.*
+
+How it works
+------------
+
+.. figure:: https://cloud.githubusercontent.com/assets/10932229/14975430/ea9c25de-1144-11e6-8824-531c81976160.png
+   :width: 350px
+   :alt: projection
+   :align: right
+   :figclass: align-center
+
+   projections (Fig. 2 of [1])
+
+A projection of 3D Newton spheres along the :math:`x` axis yields a compact 1D function:
+
+.. math::
+
+ L(z, u) = \sum_k \sum_\ell P_\ell(u)P_\ell\left(\frac{z}{r_k}\right) \frac{\prod_{r_k}(z)}{2r_k} p_{\ell k}
+
+with :math:`u = \cos(\theta)`. This function constitutes a system of equations
+expressing :math:`L(z, u)` as a linear combination of :math:`P_\ell(z/r_k)`. There
+exists for a given base a unique set of coefficients :math:`p_{\ell k}` 
+producing a least-squares fit to the function :math:`L(z, u)`.
+
+|
+|
+
+
+
+[ *extract of a comment made by Thomas Gerber (method author)* ]
```

Comment:
you can include a hyperlink to the actual comment if you like. (See readme.rst for some examples of how to include an external hyperlink in .rst)